core/io/multiplayer_api.cpp

```diff
@@ -324,6 +348,7 @@ void MultiplayerAPI::_process_rset(Node *p_node, const StringName &p_name, int p
 
 void MultiplayerAPI::_process_simplify_path(int p_from, const uint8_t *p_packet, int p_packet_len) {
 
+	ERR_EXPLAIN("Invalid packet recevied. Size too small.");
```

Comment:
recevied ->received, there are multiple instances of this typo, just do a replace :)

Reply:
Eheh, copy-paste is okay... unless you make a typo the first time xD